DESCRIPTION

```diff
@@ -0,0 +1,9 @@
+Package: datacurator
+Version: 0.1.0
+Title: Sage Bionetwork's Metadata Ingress Shiny App
+Description: This Shiny app allows users to validate metadata files and upload to Synapse.
+Authors: Rongrong Chai, Xengie Doan, Milen Nikolov, Sujay Patil, Robert Allaway, Bruno Grande, Anthony Williams, Lauren Wolfe
```

Comment:
Small change here but can you please update my name: Lauren -> Loren

Reply:
yes of course